src/Makefile

```diff
@@ -66,27 +66,29 @@ CXX=clang++
 AR=libtool -static -o
 INCS=
 
+BREW_PREFIX=$(shell which brew)
+PORT_PREFIX=$(shell which port)
```

Comment:
I am not on MacOS to test, but before PORT_PREFIX was `/opt/local` now will become path to the `port` binary, I don't know where it is, but I imagine it would be `/opt/bin/port`. What `shell which port` actually returns? Is it `/opt/local` ?

Reply:
Why are we calling it PREFIX if it is the path to the package manager executable?

Reply:
Ah I got, so the wanted behavior was really the executable path, `_PREFIX` tricked me thinking it would need to point to a directory, not an executable.

Reply:
It's just that "prefix" is something in "configure" and "makefile" jargon that is supposed to mean the base directory on top of which you will end up installing the products of the source tree. Hence Eduardo's comment.